doc/devel/coding_guide.rst

```diff
@@ -165,6 +165,48 @@ C/C++ extensions
 
 
 
+PR Review guidelines
+====================
+
+* We run on a 'use your judgement' system (for better or worse) and do
+  not have a strict nose-count rule.  If you have a commit bit, then
```

Comment:
It appears that we lost what waffle stuff we had when wafflebot got turned off.  I poked at it a bit and the events it will respond to is very limited and it looks like the new review tools do not have public web-hooks yet (so in principle we could auto-flag PRs when someone does an 'approve' review).

I think automatic labels > 'hand edited' titles > hand edited labels.